Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2221,11 +2221,12 @@ def prepare(self, doChecking = False):
         # everything else
         #
         # FIXME: remove when no longer needed
-        if "RECO" in self.stepMap or "RAW2RECO" in self.stepMap:
-                self.pythonCfgCode += "\n# Add early deletion of temporary data products to reduce peak memory need\n"
-                self.pythonCfgCode += "from Configuration.StandardSequences.earlyDeleteSettings_cff import customiseEarlyDeleteForRECO\n"
-                self.pythonCfgCode += "process = customiseEarlyDeleteForRECO(process)\n"
-                self.pythonCfgCode += "# End adding early deletion\n"
+        self.pythonCfgCode += "\n# Add early deletion of temporary data products to reduce peak memory need\n"
+        self.pythonCfgCode += "from Configuration.StandardSequences.earlyDeleteSettings_cff import customiseEarlyDeleteForRECO\n"
+        self.pythonCfgCode += "process = customiseEarlyDeleteForRECO(process)\n"
```

Comment:
since "if RECO" is removed, it's worth changing this to "customiseEarlyDelete" without "ForRECO"